src/http.rs

```diff
@@ -61,3 +72,59 @@ fn add_auth_headers<'a>(headers: &'a mut HeaderMap, user: &GlobalUser) {
         }
     }
 }
+
+////---------------------------NEW API CLIENT CODE---------------------------////
+pub fn api_client(
```

Comment:
boy do i not want to bikeshed this, but i might suggest lending some specificity for future proofing, like `cf_v4_api_client` or something. non-blocking

Reply:
ooh, i do like this idea... i defs want us to think about this more as a team. Mind if we make a separate ticket for this?

Reply:
yes, though being a soon-to-be ubiquitous public function it should be done sooner than later.